maidr/patch/heatmap.py

```diff
@@ -13,12 +13,16 @@ def heat(wrapped, _, args, kwargs) -> Axes | AxesImage:
     # Check for additional label used by MAIDR heatmap.
     fill_label = kwargs.pop("fill_label", "Fill")
 
+    _fmt = ".2f"
+    if "fmt" in kwargs:
+        _fmt = kwargs.get("fmt")
```

Comment:
Instead of checking for the value, you can try to set `fmt` to default value. Check what is the default value of `fmt` used by the `format()`, and set it here like `fmt = kwargs.get("fmt", <default value>)`.

Reply:
@SaaiVenkat Default Value of format is empty string `""`. I have added the same to the patch.

```python
fmt = kwargs.get("fmt", "")
```